mtags/src/main/scala-2/scala/meta/internal/pc/completions/SbtLibCompletions.scala

```diff
@@ -0,0 +1,86 @@
+package scala.meta.internal.pc.completions
+
+import scala.meta.internal.mtags.CoursierComplete
+import scala.meta.internal.pc.MetalsGlobal
+
+import org.eclipse.{lsp4j => l}
+
+trait SbtLibCompletions {
+  this: MetalsGlobal =>
+
+  object SbtLibExtractor {
+    def unapply(path: List[Tree]): Option[(Position, String)] = {
+      path match {
+        // "group@@" %%
+        case (lt @ Literal(Constant(group: String))) :: Select(
+              _,
+              percent1: Name
+            ) :: _ if Set("%", "%%").contains(percent1.decoded) =>
```

Comment:
We could extract `val isPercentage = Set("%", "%%")`
```suggestion
            ) :: _ if isPercentage(percent1.decoded) =>
```
the same below.